### DISCUSSION TITLE
Questions about the Type Ia SNe outputs

### DISCUSSION THREAD

**Discussion by brettppark** [TeamCOMPAS/COMPAS#1249] [Q&A] [answered]
Hello Professors and doctors in COMPAS,

I was running COMPAS to check whether this code can produce type Ia SNe.
Some src code files (i.e. COWD, WhiteDwarfs) contain codes for SN Ia but my binary evolution results don't produce type Ia SNe. They produced only for the ECSN and CC SN. (By the way i ran about 100million binaries)
Is this maybe from the problem that criteria  for type Ia SN  (mass accretions from donor, mass accumulations, the way they rich to the chandrasekhar mass, etc..) is quite ambiguous?  Or ECSN happens before SN Ia occurs because ECSN has similar criteria with SN Ia?

If you could provide some example conditions of binary systems that produce Type Ia supernovae, I would be pleased to test them on my own.

King regards,
Brett

**Comment by ilyamandel** [comment, marked as answer]
Dear Brett,

Sorry for the delayed response.  Have you included the options --evolve-double-white-dwarfs and --emit-gravitational-radiation in your runs?

If you have, please report this as an issue at https://github.com/TeamCOMPAS/COMPAS/issues, providing the exact version number and command line that you used, so we can reproduce and investigate this problem.

Best wishes,
Ilya
